port/common/omrportcontrol.c

```diff
@@ -72,8 +72,39 @@ omrport_control(struct OMRPortLibrary *portLibrary, const char *key, uintptr_t v
 			return (int32_t)PPG_mem_mem32_subAllocHeapMem32.suballocator_commitSize;
 		}
 		return 0;
+	} else if (!strcmp(OMRPORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE, key)) {
+		if (0 != value) {
+			/* IncrementSize is immutable. It can only be set once. */
+			if (0 == PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize) {
+				/* Round up the increment size to the page size and set it to global variable */
+				uintptr_t pageSize = portLibrary->vmem_supported_page_sizes(portLibrary)[0];
+				uintptr_t roundedIncrementSize = pageSize * (value / pageSize);
+				if (roundedIncrementSize < value) {
+					roundedIncrementSize += pageSize;
+				}
+				PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize = roundedIncrementSize;
+			} else {
+				return 1;
+			}
+		} else {
+			return (int32_t)PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize;
+		}
+		return 0;
 	}
-#endif
+#if defined(LINUX)
+	else if (!strcmp(OMRPORT_CTLDATA_ALLOCATE32_QUICK_ALLOC, key)) {
+		if (FALSE == value) {
+			/* QuickAlloc is immutable. It can only be set once. */
+			if (TRUE == PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc) {
```

Comment:
IMO,. comparison with `TRUE` is a sort of anti-pattern. That field should be of type `BOOLEAN` which makes a simple reference reasonable:
```c
			if (PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc) {
```

Reply:
Agreed.